### PR TITLE
feat: prepared the calendar for connecting to the API

### PR DESCRIPTION
**Pull request by eugenederii**

### CODE REVIEW

File: src/modules/schedule/components/calendar/calendar.scss [hunk 1]
```diff
@@ -11,13 +11,22 @@
   border-radius: variables.$border-radius-medium;
 }
 
+// висота row
```
Comment (IhorBalan) [inline src/modules/schedule/components/calendar/calendar.scss:14]: Only english

File: src/modules/schedule/components/calendar/calendar.scss [hunk 2]
```diff
@@ -38,8 +47,20 @@
...
+  //   border-top: none;
+  // }
+
+  // .fc-timegrid-slot-label:nth-child(odd) {
+  //   border-top: 1px solid #000;
+  // }
+
+  // .fc-timegrid-slot-minor {
+  //   border-top-style: none;
+  // }
```
Comment (IhorBalan) [inline src/modules/schedule/components/calendar/calendar.scss:60]: If we don't use the code remove it or add TODO if you will definetly need it in the future.

File: src/modules/schedule/components/calendar/calendar.tsx [hunk 3]
```diff
@@ -1,19 +1,72 @@
...
+//     end: new Date(2024, 2, 4, 14, 0, 0),
+//   },
+//   {
+//     id: '4',
+//     title: 'Событие',
+//     start: new Date(2024, 2, 5, 15, 0, 0),
+//     end: new Date(2024, 2, 5, 16, 0, 0),
+//   },
+// ];
+
```
Comment (IhorBalan) [inline src/modules/schedule/components/calendar/calendar.tsx:31]: TODO or remove the code.

File: src/modules/schedule/components/calendar/calendar.tsx [hunk 4]
```diff
@@ -1,19 +1,72 @@
...
+          return `${data.date.day} ${monthName}`;
+        }}
+        slotLabelFormat={{
+          hour: 'numeric',
+          minute: '2-digit',
+          omitZeroMinute: false,
+          meridiem: false,
+        }}
+        slotLabelContent={({ date }) => {
+          return (
```
Comment (IhorBalan) [inline src/modules/schedule/components/calendar/calendar.tsx:57]: You may omit return here.

**Review by IhorBalan — APPROVED** [review]

**Review by VadimTarasenko — APPROVED** [review]

**Pull request merged**